src/main/scala/com/fulcrumgenomics/umi/FilterConsensusReads.scala

```diff
@@ -92,17 +94,28 @@ class FilterConsensusReads
   @arg(flag='o', doc="Output SAM or BAM file.") val output: PathToBam,
   @arg(flag='r', doc="Reference fasta file.") val ref: PathToFasta,
   @arg(flag='R', doc="Reverse [complement] per base tags on reverse strand reads.") val reversePerBaseTags: Boolean = false,
-  @arg(flag='M', minElements=1, maxElements=3, doc="The minimum number of reads supporting a consensus base/read.")
+
+ // Filters that are applied both per-base and per-read
+  @arg(flag='M', group="Per-read/Per-base", minElements=1, maxElements=3, doc="The minimum number of reads supporting a consensus base/read.")
```

Comment:
Either this should not be in a group, or `--min-mean-base-quality-score` should be in a group I think.  I'm not sure I'm a fan of the grouping here in general though - when we have to create 4 groups to accurately group ~12 options.

Reply:
Also, if you do keep the groups I think I would add " Filters" to the ends of the group names.